cmd/subcommands/keys.go

```diff
@@ -41,7 +41,8 @@ var (
 // getPassphrase fetches the correct passphrase depending on if a file is available to
 // read from or if the user wants to enter in their own passphrase. Otherwise, just use
 // the default passphrase.
-func getPassphrase() (string, error) {
+// argument passphraseconfirmation is true when confirmation is required false otherwise 
+func getPassphrase(passphraseconfirmation bool) (string, error) {
```

Comment:
Don't make a function with a bool parameter. instead make two functions that do what you want, that is, `getPassphraseWithConfirm`, `getPassphrase` (assume no confirm on that one)

Reply:
@fxfactorial just made the change as suggested. Thanks